cmd/build/v2/servicecontext.go

```diff
@@ -0,0 +1,38 @@
+package v2
+
+import (
+	"github.com/okteto/okteto/pkg/filesystem"
+	oktetoLog "github.com/okteto/okteto/pkg/log"
+	"github.com/okteto/okteto/pkg/model"
+	"github.com/okteto/okteto/pkg/repository"
+)
+
+type serviceContextInterface interface {
+	IsCleanBuildContext() bool
+}
+
+type serviceContext struct {
+	isClean bool
+}
+
+func (sc *serviceContext) IsCleanBuildContext() bool {
+	return sc.isClean
+}
+
+func getServiceContext(buildInfo *model.BuildInfo) *serviceContext {
+	wdCtrl := filesystem.NewOsWorkingDirectoryCtrl()
```

Comment:
Should we reuse the one from the build? That way we can test from an integration test if the `wd` is the same

Reply:
c4ade2c07260cf3dd979a7e0c69bc6bcda87c71d